src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -284,6 +311,18 @@ Copyright (c) .NET Foundation. All rights reserved.
     </FindInList>
   </Target>
 
+  <!--
+    Computes the applicable @(ResolvedRazorComponentConfiguration) that match the project's configuration.
+  -->
+  <Target
+    Name="ResolveRazorComponentConfiguration"
+    Condition="'$(RazorDefaultComponentConfiguration)'!=''">
+
+    <ItemGroup>
+      <ResolvedRazorComponentConfiguration Include="@(RazorComponentConfiguration->WithMetadataValue('Identity', '$(RazorDefaultComponentConfiguration)')->Distinct())" />
```

Comment:
Thoughts? We basically get 2 configurations in a single build - one for RazorGenerate and one for RazorComponent items

Reply:
Yeah I guess I was hoping we could do without the need for a separate configuration. I was hoping that components would be more first class. 

A RazorConfiguration in VS is like a version and there's one configuration per-project. I'd prefer to have a single configuration in each project, and this would require a lot of tooling change. 

Could we structure the command line args passed to the compiler such that a piece of metadata identifies what is a component? Or maybe we could attach a `Kind` metadata to each file.